[5.11.0] Update sections with callback regex configuration (#3715)
wso2/docs-is: en/docs/learn/self-registration-and-account-confirmation.md

    </div></div> 
    </div></td>
    </tr>
    <tr class="even">
    <td>User self registration callback URL regex</td>

>     <td>The regex pattern used to validate the callback URL you are adding as the recovery flow callback url. 

Reviewer: ```suggestion
    <td>The regex pattern used to validate the callback URL you are adding as the recovery flow callback URL. 
```

    
    <div class="admonition note">
    <p class="admonition-title">Note</p>

>     <p>The sample <b>User self registration callback URL regex</b> to use when testing the product is <code>^https:\/\/localhost:9443\/.*</code>. But user should modify it meet their requirements when they are deploying the product.</td></p>

Reviewer: ```suggestion
    <p>The sample <b>self registration callback URL regex</b> to use when testing the product is <code>^https:\/\/localhost:9443\/.*</code>. However, users should modify it to meet their requirements when they deploy the product.</td></p>
```
